inheritance/.cs50.yml

```diff
@@ -2,3 +2,9 @@ lab50:
   files:
     - !exclude "*"
     - !include inheritance.c
+
+check50:
+  files:
+    - !exclude "*"
+    - !include inheritance.c
+    - !include testing.c
```

Comment:
The `.yml` file should only include files that students submit, not the ones that we need to include. For that, you should add a line like https://github.com/cs50/problems/blob/2020/fall/runoff/__init__.py#L10 into the `exists` function.